karma.conf.js

```diff
@@ -9,14 +9,14 @@ module.exports = function (config) {
         basePath: '',
         // frameworks to use
         // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
-        frameworks: ['jasmine-jquery', 'jasmine'],
+        frameworks: ['jasmine'],
         // list of files / patterns to load in the browser
         files: ['tests_index.js'],
 
         // preprocess matching files before serving them to the browser
         // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
         preprocessors: {
-            'tests_index.js': ['webpack', 'sourcemap', 'coverage'],
+            'tests_index.js': ['webpack', 'sourcemap'],
```

Comment:
Yes! We still do, but instead of preprocessing with this karma plugin that is unmaintained and has many security issues we do it with a newer babel plugin.

I still think we should move out of Karma soon, but that should be for v4! Also grunt, we need to get out of it ASAP